components/ColorMixLabel/ColorMixLabel.module.css

```diff
@@ -0,0 +1,20 @@
+.colorMixLabelWrapper {
+  position: relative;
+  margin: 0 auto;
+  background-color: var(--gray-3);
+  padding: 0.5rem 1rem;
+  border-radius: 8px;
+}
+
+.closeButton {
```

Comment:
Add a set height + width, so the button is a perfect circle

Reply:
Temp button since we will be using the cross-svg from Figma